src/main/java/edu/kit/ipd/crowdcontrol/objectservice/notification/Notification.java

```diff
@@ -97,7 +97,7 @@ public void run() {
         try {
             policy.invoke(this);
         } catch (RuntimeException e) {
-            LOGGER.error("An Exception occurred while checking or sending a notification.", e);
+            LOGGER.error("An exception occurred while checking or sending a notification.", e);
```

Comment:
this line printent stack traces before, you just changed Exception to exception ...